docs/application/dotnet/guides/nui/flex-layout.md

```diff
@@ -109,9 +109,76 @@ var flexLayout = new FlexLayout();
+| `FlexGrow`             | float           | The flex grow describes how any space within a flex layout should be distributed among its children along the main axis. |
+
+## FlexAlignmentSelf property
+
+The `FlexAlignmentSelf` property has the same options and effect as `Alignment` property, but instead of affecting the children within a flex layout, This property is applied to a single child to change its alignment within its parent. The align self overrides any option set by the parent with align items.
+
+The following figure shows how to work the `FlexAlignmentSelf` property for item `2`:
+
+![FlexAlignmentSelf](./media/flexAlignmentSelf.png)
+
+## FlexPositionType property
+
+The `FlexPositionType` property of an element defines how it is positioned within its parent. By default, an element is positioned relatively. This means an element is positioned according to the normal flow of the layout, and then offset relative to that margin based on the values of start, end, top, and bottom.
+The offset does not affect the position of any sibling or parent elements. When positioned absolutely, an element doesn't take part in the normal layout flow. It is instead laid out independently of its siblings. The position is determined based on the position property.
+
+The following figure shows how to work the `FlexPositionType` property for item `2`:
+
+![FlexPositionType](./media/flexPositionType.png)
+
+## FlexAspectRatio property
+
+The `FlexAspectRatio` property has the following properties:
+
+ - Accepts any floating-point value greater than `0`. The default value is undefined.
+ - Defined as the ratio between the width and the height of a node, e.g. if a node has an aspect ratio of `2`, then its width is twice the size of its height.
+ - Has a higher priority than `FlexGrow` property.
+ - If aspect ratio, width, and height are set then the cross axis dimension is overridden.
+
+## FlexBasis property
+
+The `FlexBasis` property is an axis-independent way of providing the default size of an item along the main axis.
+Setting the flex basis of a child is similar to setting the width of that child if its parent flex layout is set flex direction to `Row` or setting the height of a child if its parent flex layout set flex direction to `Column`.
+
+The flex basis of an item is the default size of that item, the size of the item before any flex grow and flex shrink calculations are performed.
+
+## FlexShrink property
+
+The `FlexShrink` describes how to shrink children along the main axis in the case that the total size of the children overflows the size of the flex layout on the main axis.
+
+The flex shrink accepts any floating point value greater or equal to `0`. The default value is `1`.
+
+The following figure shows how to work the `FlexShrink` property for item `2`:
```

Comment:
```suggestion
The `FlexShrink` specifies how to shrink the children along the main axis in case, the total size of the children overflows the size of the flex layout on the main axis. The flex-shrink accepts any floating-point value greater or equal to `0`. The default value is `1`.

The following figure shows how to use the `FlexShrink` property: